Frameworks/UIKit/StarboardXaml/CompositorInterface.mm

```diff
@@ -332,7 +333,12 @@ void SetParams(UIFont* font,
         _isBold = (mask & UIFontDescriptorTraitBold) > 0;
         _isItalic = (mask & UIFontDescriptorTraitItalic) > 0;
         std::wstring wideBuffer = Strings::NarrowToWide<std::wstring>(text);
-        ConstructGlyphs([[font fontName] UTF8String], wideBuffer.c_str(), wideBuffer.length());
```

Comment:
Why not font fontFamilyName?  We should be returning the right family name there.